doc/examples/sprites/lib/generated_plugin_registrant.dart

```diff
@@ -0,0 +1,16 @@
+//
+// Generated file. Do not edit.
```

Comment:
What is this file?

Reply:
I think this is a new file that all flutter projects have now but these old examples built with older flutter didn't. not sure if we should version or not but it doesn't say on the file it shouldn't be. I would look at the other newer examples to see what we do. maybe the new flutter create adds it to the gitignore.

Reply:
I just checked, it is on git ignore, I am going to remove it